### PR TITLE
AI Assistant: extend blocks toolbar

### LINKED ISSUE #31321: AI Assistant: Add toolbar option in paragraph block
Add the toolbar option for one or multiple paragraph blocks.

This will include the dropdown list empty since they will be added in follow-up issues.

Depends on #31317

### LINKED ISSUE #31323: AI Assistant:  Add correct spelling and grammar action in paragraph extension
Introduce the `Correct Spelling and Grammar` option in paragraph AI toolbar option.

Depends on #31321

### LINKED ISSUE #31324: AI Assistant:  Add simplify action in paragraph extension
Introduce the `Simplify` option in paragraph AI toolbar option.

Depends on #31321

### PR DESCRIPTION
**Pull request by retrofox**
Fixes https://github.com/Automattic/jetpack/issues/31324
Fixes https://github.com/Automattic/jetpack/issues/31323
Fixes https://github.com/Automattic/jetpack/issues/31321

## Proposed changes:

* AI Assistant: extend blocks toolbar

### Other information:

- [x] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to the block editor
* Create a paragraph and heading block
* Select the block
* Confirm you see the AI Assistant dropdown in the block toolbar 

Heading (h4) | Paragraph
-------|-------|
Screenshot 2023-06-13 at 11 28 11 | Screenshot 2023-06-13 at 11 28 35

* Confirm it shows a log line in the dev console when clicking on the items

image

### CODE REVIEW

**Comment by lhkowalski** [comment]
Will this solve #31321?

**Comment by retrofox** [comment]
> Will this solve #31321?

yes, let me link the issues

File: projects/plugins/jetpack/changelog/update-ai-assistant-add-quick-edits-controls [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+AI Assistant: extened blocks toolbar
```
Comment (dhasilva) [inline projects/plugins/jetpack/changelog/update-ai-assistant-add-quick-edits-controls]:
```suggestion
AI Assistant: extend blocks toolbar
```

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-controls/index.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+import { post, termDescription } from '@wordpress/icons';
+import React from 'react';
+/**
+ * Internal dependencies
+ */
+import AIAssistantIcon from '../../icons/ai-assistant';
+import './style.scss';
+
+// Quick edits option: "Correct spelling and grammar"
+const QUICK_EDIT_KEY_CORRECT_SPELLING = 'correct-spelling' as const;
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-controls/index.tsx:20]: TIL about the `as const` :clap:

Comment (retrofox) [reply projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-controls/index.tsx:20]: it's a way to combine constants with types

**Review by dhasilva — APPROVED** [review]
LGTM :rocket:

**Pull request merged**